site/en/docs/extensions/mv3/declare_permissions/index.md

```diff
@@ -129,348 +122,239 @@ If your extension needs to run on `file://` URLs or needs to operate in incognit
+
+`"platformKeys"`
+: Gives access to the <a href="/docs/extensions/reference/platformKeys/">chrome.platformKeys</a> API.
+
+`"power"`
+: Gives access to the <a href="/docs/extensions/reference/power/">chrome.power</a> API.
+
+`"printerProvider"`
+:Gives access to the <a href="/docs/extensions/reference/printerProvider/">chrome.printerProvider</a> API.
+
+`"printing"`
+: Gives access to the <a href="/docs/extensions/reference/printing/">chrome.printing</a> API.
+
+`"printingMetrics"`
+: Gives access to the <a href="/docs/extensions/reference/printingMetrics/">chrome.printingMetrics</a> API.
+
+`"privacy"`
+: Gives access to the <a href="/docs/extensions/reference/privacy/">chrome.privacy</a> API.
+
+`"processes"`
+: Gives access to the <a href="/docs/extensions/reference/processes/">chrome.processes</a> API.
+
+`"proxy"`
+: Gives access to the <a href="/docs/extensions/reference/proxy/">chrome.proxy</a> API.
+
+`"scripting"`
+: Gives access to the <a href="/docs/extensions/reference/scripting/">chrome.scripting</a> API.
+
+`"search"`
+: Gives access to the <a href="/docs/extensions/reference/search/">chrome.search</a> API.
+
+`"sessions"`
+: Gives access to the <a href="/docs/extensions/reference/sessions/">chrome.sessions</a> API.
+
+`"sidePanel"`
+: Gives access to the <a href="/docs/extensions/reference/sidePanel/">chrome.sidePanel</a> API.
+
+`"storage"`
+: Gives access to the <a href="/docs/extensions/reference/storage/">chrome.storage</a> API.
+
+`"system.cpu"`
+: Gives access to the <a href="/docs/extensions/reference/system_cpu/">chrome.system.cpu</a> API.
+
+`"system.display"`
+: Gives access to the <a href="/docs/extensions/reference/system_display/">chrome.system.display</a> API.
+
+`"system.memory"`
+: Gives access to the <a href="/docs/extensions/reference/system_memory/">chrome.system.memory</a> API.
+
+`"system.storage"`
+: Gives access to the <a href="/docs/extensions/reference/system_storage/">chrome.system.storage</a> API.
+
+`"tabCapture"`
+: Gives access to the <a href="/docs/extensions/reference/tabCapture/">chrome.tabCapture</a> API.
+
+`"tabGroups"`
+: Gives access to the <a href="/docs/extensions/reference/tabGroups/">chrome.tabGroups</a> API.
+
+`"tabs"`
+: Gives access to privileged fields of the <a
+          href="/docs/extensions/reference/tabs#type-Tab"><code>Tab</code></a> objects used by several APIs,
+including <a href="/docs/extensions/reference/tabs">chrome.tabs</a> and <a href="/docs/extensions/reference/windows">chrome.windows</a>. You usually won't need to declare this permission to use those APIs.
+
+`"topSites"`
+: Gives access to the <a href="/docs/extensions/reference/topSites/">chrome.topSites</a> API.
+
+`"tts"`
+: Gives access to the <a href="/docs/extensions/reference/tts/">chrome.tts</a> API.
+
+`"ttsEngine"`
+: Gives access to the <a href="/docs/extensions/reference/ttsEngine/">chrome.ttsEngine</a> API.
+
+`"unlimitedStorage"`
```

Comment:
@oliverdunk I was under the impression that `unlimitedStorage` only worked with `chrome.storage.local` not sync, managed or sessions...

I noticed this also implied in https://developer.chrome.com/docs/extensions/mv3/storage-and-cookies/#storage-persistence

Did this change at some point?

Reply:
Oh interesting, I hadn't given that thought but it looks like you're right - this applies to only `chrome.storage.local` (which makes sense). It also applies to web storage like IndexedDB, where it also exempts that storage from eviction.

We should probably rewrite this with that in mind :)

Reply:
@oliverdunk see the line below.

Reply:
I don't think this says it is limited to chrome.storage.local, right? What we have now ("an unlimited quota for chrome.storage") feels misleading even if we do have clarification.

Reply:
Sorry. Next two lines. It says:

"Unlimited quota for chrome.storage, IndexedDB, Cache Storage, and Origin Private File System.

Reply:
That's still wrong though, right? There is not "unlimited quota for chrome.storage", only `chrome.storage.local`.

Reply:
>It also applies to web storage like IndexedDB, where it also exempts that storage from eviction.

Please list all types of memory that `unlimitedStorage` applies to.

Reply:
IndexedDB, Cache Storage and WebSQL.

Reply:
so the change here is to replace chrome.storage and Origin Private File System with WebSQL?

Reply:
I would say:

- Add WebSQL to list including Origin Private File System (apologies, I forgot about that one, but it looks like it would apply).
- Replace `chrome.storage` with `chrome.storage.local`

Reply:
Looks good.

Reply:
ok! What links do I need to add?

Reply:
- As Oliver said, change `chrome.storage` to `chrome.storage.local` and change the link to this: https://pr-7249-static-dot-dcc-staging.uc.r.appspot.com/docs/extensions/reference/storage/#property-local
- Remove the links to IndexedDB and Cache Storage.

@oliverdunk, can we have a sidebar discussion about WebSQL? It's deprecated, but some reason, I have it in the back of my head that this is sticking around for the extensions universe. I'd like to let Alex finish this PR, and it'll take me all of five minutes to fix this later.

Reply:
> Remove the links to IndexedDB and Cache Storage.

I'm not sure I agree with this? It affects both of those APIs.

> @oliverdunk, can we have a sidebar discussion about WebSQL? It's deprecated, but some reason, I have it in the back of my head that this is sticking around for the extensions universe.

For sure - TLDR, it's going away for everyone. There's currently a deprecation trial to get it back if you need it but that won't last forever. I called it out for completeness but could also see an argument not to document it.

Reply:
@oliverdunk I am super confused. Can you please just give us a list of every API covered by `"unlimitedStorage"` unlimited storage?

Reply:
`chrome.storage.local`, IndexedDB, Cache Storage, Origin Private File System and WebSQL.

Reply:
As Oliver and I discussed yesterday, we need to leave WebSQL off the list. It's deprecated. We don't want anyone doing anything new with it.